actions/knowledge/knowledge.ts

```diff
@@ -0,0 +1,36 @@
+'use server';
+import path from 'path';
+import { exec } from 'child_process';
+import { promisify } from 'util';
+
+const execPromise = promisify(exec);
+
+export async function ingest(
```

Comment:
This is where we run knowledge ingest command on server side to ingest file on fly.